lib/client/Client.js

```diff
@@ -650,7 +650,7 @@ export default class Client {
     return {
       Accept: 'application/json; charset=UTF-8',
       'Content-Type': 'application/json; charset=UTF-8',
-      Authorization: this.authToken
+      Authorization: 'X-JMAP ' + this.authToken
```

Comment:
Yes, I already expected this concern :-)

And yes, you're right, the `X-JMAP` scheme is only required "If the user authenticated via the JMAP auth protocol". I agree to your suggestion to therefore only send that scheme when initialized using the `withAuthAccess` method. So besides the `authToken` we should also store the `authScheme` as a property of the client instance and use that for composing the Authorization header. To make it complete, I suggest to extend the `withAuthenticationToken` method with an additional parameter stating the `authScheme` assigned to the given token.

I'll update the pull request if you agree.